src/components/MainContent.tsx

```diff
@@ -140,12 +143,19 @@ export default function MainContent(props: IMainContentProps): JSX.Element {
                       <Form.Control
                         type="text"
                         placeholder="Player name"
-                        size="sm"
-                        className={`my-1 ${dark ? "dark-input" : ""}`}
-                        defaultValue={player.name}
+                        size="md"
```

Comment:
md not an available size, sm or lg reference (https://react-bootstrap.github.io/components/forms/#form-control-props)